src/Microsoft.AspNet.Identity/LockoutOptions.cs

```diff
@@ -10,7 +10,7 @@ public class LockoutOptions
         /// <summary>
         ///     If true, will enable user lockout when users are created
         /// </summary>
-        public bool EnabledByDefault { get; set; } = false;
+        public bool AllowedForNewUsers { get; set; } = false;
```

Comment:
It seems I am a bit lost with one thing here: aren't we supposed to flip the default for this? If yes, where is that flip happening?